src/Mod/Part/App/PrimitiveFeature.cpp

```diff
@@ -879,12 +880,15 @@ Spiral::Spiral(void)
     Radius.setConstraints(&quantityRange);
     ADD_PROPERTY_TYPE(Rotations,(2.0),"Spiral",App::Prop_None,"The number of rotations");
     Rotations.setConstraints(&quantityRange);
+    ADD_PROPERTY_TYPE(SegmentLength,(1.0),"Spiral",App::Prop_None,"The number of turns of an spiral segment");

```

Comment:
Unrelated: do these strings that look like 'tooltips' get translated somewhere?

Reply:
Yep: they appear to be translated at point of use, e.g. https://github.com/FreeCAD/FreeCAD/blob/20c0f14efc5ebcd0722a9605140bb0686b3c2eaa/src/Gui/propertyeditor/PropertyItem.cpp#L335

Reply:
I don't know anything about translation mechanism ...